concent-builder/configure.yml

```diff
@@ -8,45 +8,8 @@
     - become:      yes
       become_user: root
       block:
-        - name:    Install required packages for apt_key and apt_repository
-          apt:  name={{ item }}
-          with_items:
-            - apt-transport-https
```

Comment:
This commit only removes this code. It should also be adding the new code in the new place.